app/views/layouts/application.html.haml

```diff
@@ -1,7 +1,5 @@
-- if @mks || @vmrc || @vnc || controller.action_name == "menutest"
-  = yield
```

Comment:
Do we want to wrap console sessions in our full header/footer layout?  They currently open as independent popup windows.  This will result in duplicate sets of html, header, body, footer tags in the resulting markup.  See app/views/vm_common/console_*.html.haml files [1] in the diff.

[1] https://github.com/ManageIQ/manageiq/pull/4139/files#diff-4c8342ed474fde5dcb87be9ae7d96eda